src/Be.Vlaanderen.Basisregisters.Api/Search/Pagination/ExtractPaginationRequestExtension.cs

```diff
@@ -11,23 +11,34 @@ public static class ExtractPaginationRequestExtension
         public static IPaginationRequest ExtractPaginationRequest(
             this HttpRequest request,
             int defaultOffset = 0,
-            int defaultLimit = 100, 
+            int defaultLimit = 100,
             int maxLimit = 500)
         {
             var pagination = request.Headers[AddPaginationExtension.HeaderName];
 
             if (pagination == new StringValues(NoPagination))
+            {
                 return new NoPaginationRequest();
+            }
 
             var offset = defaultOffset;
             var limit = defaultLimit;
 
+            if (int.TryParse(request.Query["offset"], out var newOffset))
+            {
+                offset = newOffset;
+            }
+
+            if (int.TryParse(request.Query["limit"], out var newLimit))
+            {
+                limit = newLimit;
+                limit = Math.Min(limit, maxLimit);
+            }
+
```

Comment:
This is checking the request `Query` while L#17 uses headers.
Maybe we should pair and see where the problem lies?